### PR TITLE
Allow passing label values as keyword arguments

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by brian-brazil** [comment]
This is getting a bit complicated. I think that if we're going to this this then the ability to pass a dict as the first arg should be deprecated. I think there's only one user of that out there.

**Comment by alex** [comment]
I have no objection to that; I find the "raw dict" UX a bit unpleasant myself (hence this PR :-)). Is there a precedent for how deprecations are handled in this library?

**Comment by brian-brazil** [comment]
This would be the first deprecation. Probably best to reach out to users and see if they can take a straight breakage.

**Comment by alex** [comment]
Is there a particular place to do that?

**Comment by brian-brazil** [comment]
You could hit the mailing list, or ping @korfuri

**Comment by alex** [comment]
Sorry it took me a while, mailing list messaged: https://groups.google.com/forum/#!topic/prometheus-developers/KTAvr3DdVi4

**Comment by alex** [comment]
@brian-brazil Been about 5 days and only one person has chimed in. Shall I update this PR to remove the dict-passing API?

**Comment by brian-brazil** [comment]
Sounds like a plan

**Comment by alex** [comment]
Great; done!

File: README.md [hunk 1]
```diff
@@ -185,7 +185,16 @@ c.labels('get', '/').inc()
...
+Labels can also be passed as keyword-arguments:
+
+```python
+from prometheus_client import Counter
+c = Counter('my_requests_total', 'HTTP Failures', ['method', 'endpoint'])
+c.labels(method='get', endpoint='/').inc()
+c.labels(method='post', endpoint='/submit').inc()
+```
+
+Or as a dict:
```
Comment (brian-brazil) [inline README.md]: The example here should be removed

Comment (alex) [inline README.md]: You're looking at a previous version of the diff, it's removed on the latest.

**Comment by brian-brazil** [comment]
Thanks!

**Comment by alex** [comment]
Thank you!

**Pull request merged**